internal/pkg/warewulfd/warewulfd.go

```diff
@@ -21,30 +20,28 @@ func RunServer() error {
 		for range c {
 			err := LoadNodeDB()
 			if err != nil {
-				wwlog.Printf(wwlog.WARN, "Could not load database: %s\n", err)
+				fmt.Printf("ERROR: Could not load database: %s\n", err)
 			}
 		}
 	}()
 
 	err := LoadNodeDB()
 	if err != nil {
-		wwlog.Printf(wwlog.WARN, "Could not load database: %s\n", err)
+		fmt.Printf("ERROR: Could not load database: %s\n", err)
 	}
 
-	wwlog.Printf(wwlog.DEBUG, "Registering handlers for the web service\n")
-
 	http.HandleFunc("/ipxe/", IpxeSend)
 	http.HandleFunc("/kernel/", KernelSend)
 	http.HandleFunc("/kmods/", KmodsSend)
 	http.HandleFunc("/container/", ContainerSend)
 	http.HandleFunc("/overlay-system/", SystemOverlaySend)
 	http.HandleFunc("/overlay-runtime", RuntimeOverlaySend)
 
-	wwlog.Printf(wwlog.VERBOSE, "Starting HTTPD REST service\n")
+	daemonLogf("Starting HTTPD REST service\n")
 
 	err = http.ListenAndServe(":9873", nil)
 	if err != nil {
-		wwlog.Printf(wwlog.ERROR, "Could not start listening service: %s\n", err)
+		fmt.Printf("ERROR: Could not start listening service: %s\n", err)
 		os.Exit(1)
```

Comment:
Avoid using `os.Exit()`. This should be replaced with: 

```suggestion
		return errors.Wrap(err, "could not start listening service")
```